.github/workflows/test.yml

```diff
@@ -1,5 +1,8 @@
 name: "build"
-on: push
+on:
+  push:
+     branches: [ master ]
```

Comment:
Where do we see the results for master? It seems like the code below is only adding PR comments (`github.event_name == 'pull_request'`) so results for master will be lost?

Reply:
Differential CI needs something to compare changes against, i.e. a base branch, and this is about commenting on PRs, which doesn’t exist for pushes to master, so this stuff only happens for PRs.

You can of course always run `nix-build -A tests.perf` on any commit to get the base data, i.e. the CSV file:
```
~/dfinity/motoko $ cat $(nix-build -A tests.perf)/stats.csv
gas/assetstorage;14442
size/assetstorage;165585
gas/qr;1237055836
size/qr;220465
gas/reversi;1305950
size/reversi;160031
```

We may want to publish the data along with other reports (e.g. flamegraphs), see #2568 for that.